TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift

```diff
@@ -42,7 +44,9 @@ public struct PushFactorPayload: FactorPayload {
   ///because the systems that will process this attribute assume it is not directly identifying information.
   public let identity: String
   ///Type of the factor, push by default.
-  public let factorType: FactorType = .push
+  public let factorType: FactorType
+  /// Allow factor migration from iPhone to iPhone
+  public let allowIphoneMigration: Bool
   ///(Optional) Registration token generated by APNS when registering for remote notifications.
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/c51c4513007d818dab4b64cd25c837959c7ec03d/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L50"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>FactorPayload.swift:50</code></td>
    </tr>
  </tbody>
</table>